system_test/helpers/aest_nodes.erl

```diff
@@ -53,10 +53,19 @@
 
 %% AWK script to keep only error, critical, alert and emergency log lines with
 %% all the extra lines following the log lines.
+%% FIXME: Temporarily ignore dispatch_worker errors, remove when fixed.
```

Comment:
@sylane Before merging we shall ensure we at least track when to identify and address somehow log message.